cloud-prepare/vpc-peering.md

```diff
@@ -0,0 +1,210 @@
+# Submariner VPC Peering Support
+
+This document describes the technical details and impact of our proposal design
+to include Virtual Private Cloud (VPC) Peering support for Submariner and
```

Comment:
Note: VPC Peering is a term used by AWS and GCP, but other providers have different names for it. For e.g, the equivalent in Azure is "virtual network peering".

Reply:
Can you or someone propose a better name?

Reply:
My vote for `peering` IT is generic enough and all clouds use `peering` in their offering.

AWS: VPC Peering
GCP: VPC NEtwork Peering or Cloud Peering.
Azure: Virtual Network Peering.